integration-tests/bats/sql-merge.bats

```diff
@@ -74,22 +74,109 @@ SELECT COUNT(*) > 0 FROM test WHERE pk=3;
 SQL
     [ $status -eq 0 ]
     [[ "$output" =~ "true" ]] || false
+
+    run dolt sql -r csv -q "select count(*) from dolt_status"
+    [ "${#lines[@]}" -eq 2 ]
+    [ "${lines[1]}" = "0" ]
+
+    run dolt sql -r csv -q "select hash from dolt_branches where branch='main'"
+    MAIN_HASH=${lines[1]}
+
+    run dolt sql -r csv -q "select hash from dolt_branches where branch='feature-branch'"
+    FB_HASH=${lines[1]}
+
+    [ "$MAIN_HASH" = "$FB_HASH" ]
 }
 
 @test "sql-merge: DOLT_MERGE with autocommit off works in fast-forward." {
-     run dolt sql --disable-batch << SQL
+     run dolt sql << SQL
 set autocommit = off;
 SELECT DOLT_COMMIT('-a', '-m', 'Step 1');
 SELECT DOLT_CHECKOUT('-b', 'feature-branch');
 INSERT INTO test VALUES (3);
 SELECT DOLT_COMMIT('-a', '-m', 'this is a ff');
 SELECT DOLT_CHECKOUT('main');
 SELECT DOLT_MERGE('feature-branch');
--- commit;
 SELECT DOLT_CHECKOUT('-b', 'new-branch');
 SQL
-    skip "That commented out commit; is necessary now. It should not be."
     [ $status -eq 0 ]
```

Comment:
For these tests, if the only assertion  you're making is status == 0, then you can just remove 'run', it's equivalent (and gives better error info if it fails)